Mathlib/Algebra/Order/Module/Defs.lean

```diff
@@ -59,22 +59,32 @@ used implications are:
   * `SMulPosStrictMono → SMulPosMono`
   * `PosSMulReflectLE → PosSMulReflectLT`
   * `SMulPosReflectLE → SMulPosReflectLT`
-* When `β` is a linear order: `PosSMulStrictMono → PosSMulReflectLE`
-* When `α` is a linear order: `SMulPosStrictMono → SMulPosReflectLE`
+* When `β` is a linear order:
+  * `PosSMulStrictMono → PosSMulReflectLE`
+  * `PosSMulReflectLT → PosSMulMono`
+  * `SMulPosReflectLT → SMulPosMono`
+  * `PosSMulReflectLE → PosSMulStrictMono`
+  * `SMulPosReflectLE → SMulPosStrictMono`
```

Comment:
Can you make clear these are not instances?

Reply:
Which are not instances

Reply:
OK